.github/workflows/publish-site.yml

```diff
@@ -0,0 +1,27 @@
+name: Website Build
+on:
+  push:
+    branches:
+      - 'feature/heavy-wip-website'
+  pull_request:
+    # none
+jobs:
+  Build:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: 'true'
+    - name: Setup .NET 6.0
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 6.0.100-preview.7.21379.14
+    - name: Run Statiq
+      run: dotnet run -c Release --project src/Website/Silk.NET.Statiq/Silk.NET.Statiq.csproj
+    - name: Push to Website Branch
+      run: |
+        git config --local user.email "9011267+dotnet-bot@users.noreply.github.com"
+        git config --local user.name "The Silk.NET Automaton"
+        git add -f docs/
+        git commit -am "Push latest built in-development GitHub Pages site"
+        git push -f origin HEAD:ci/github-pages
```

Comment:
There has to be a way to not make this force pushed always. Can't we just merge this?

Reply:
I'll align the histories after merge, but during development of the website this isn't really possible without the histories going wibbly wobbly timey wimey. Will do this in a follow-up MR immediately after merge.

Reply:
There's [Action](https://github.com/marketplace/actions/deploy-to-github-pages) for it